spharpy/samplings/coordinates.py

```diff
@@ -36,7 +36,17 @@ def __init__(
         sh_order : int, optional
             Maximum spherical harmonic order of the sampling grid.
             The default is ``None``.
+        radius_tolerance : float, optional
```

Comment:
I would prefere a relative tolerance, but i would be fine with absolute as well.
we could call it radius_atol or radius_rtol, then its directly clear whch kind of tolerance it is.

Reply:
Opted to rename it to radius_atol

Reply:
sorry, I did not see the comment when reviewing and suggested to not name it atol. 
Still, I think atol is not as intuitive as simply naming it tolerance.